### DISCUSSION TITLE
Propose: shorter links?

### DISCUSSION THREAD

**Discussion by patrickdrd** [iptv-org/community#247] [Ideas]
hello,
could it be possible to make the links shorter?
I mean [xtream-editor](https://xtream-editor.com/en/editor/) supports a max of 300 chars in url addresses,
so some of them don't work and create problems instead

**Comment by BellezaEmporium** [comment]
Some simply can't be shortened because that's how the link works, and some can.

**Comment by patrickdrd** [comment]
yes, I understand, obviously, maybe a url shortener can be used in the first case?

**Comment by Dum4G** [comment]
Simply don't use editors that bad. If we shorten every link there would be thousands of raging people with awful IPTV players and dozens of unavailable links. Consider switching to m3u.in or a PC app for the purpose

**Comment by patrickdrd** [comment]
xtream editor is bad?
I wouldn't think so, I'm using it for some months with 5 lists and I'm absolutely happy with it,
and as you say, there are thousands of people, I can argue that there are thousands of lists as well,
and editors help in organizing, syncing/updating, etc,
so not using an editor would be bad eventually,

anyway, ok, I would be editing this list manually, but the sync/update of it would be some pain,
that's why I suggested this for discussion

**Comment by Dum4G** [comment]
m3u.in doesn't have that limitations

**Comment by patrickdrd** [comment]
m3u.in's interface is pretty bad though, not easy to move channels from a (large) group to another

**Comment by freearhey** [comment]
@themak1985 It is better to hide comments rather than delete them to make it easier to track repeated violations of the rules.

**Comment by themak1985** [comment]
Got it, will do @freearhey
